bots/notify/src/test/java/org/openjdk/skara/bots/notify/issue/IssueNotifierTests.java

```diff
@@ -1061,6 +1062,11 @@ void testIssueBackport(TestInfo testInfo) throws IOException {
 
             // Labels should not
             assertEquals(0, backport.labelNames().size());
+
+            // If the parent issue has a security level (can be configured when running a test manually) it should be propagated
+            if (level != null) {
+                assertEquals(level.asString(), backport.properties().get("security").asString());
+            }
```

Comment:
Maybe also worth also checking that if level is null, the returned property is empty? This could be done as a follow-up given the need to get this in right away.